library/std/src/lib.rs

```diff
@@ -215,6 +215,7 @@
 // std is implemented with unstable features, many of which are internal
 // compiler details that will never be stable
 // NB: the following list is sorted to minimize merge conflicts.
+#![cfg_attr(not(bootstrap), feature(doc_spotlight))]
```

Comment:
This seems unrelated, maybe a rebase conflict?

Reply:
Yup it is, I was waiting for CI, will fix it as when I get to my computer